filebeat/fileset/modules.go

```diff
@@ -474,10 +475,46 @@ func (reg *ModuleRegistry) LoadML(esClient PipelineLoader) error {
 	return nil
 }
 
+// SetupML sets up the machine-learning configurations into Elasticsearch using Kibana, if Xpack is available
+func (reg *ModuleRegistry) SetupML(esClient PipelineLoader, kibanaClient *kibana.Client) error {
+	haveXpack, err := mlimporter.HaveXpackML(esClient)
+	if err != nil {
+		return errors.Errorf("Error checking if xpack is available: %v", err)
```

Comment:
It seems the file already disagrees with this statement, but error strings should not be capitalized. https://github.com/golang/go/wiki/CodeReviewComments#error-strings